Decisions.md

```diff
@@ -0,0 +1,29 @@
+# Decisions
+
+## Voting
+
+Decisions are based on voting using the following:
+
+* +1 : For
+* +0: Mildly for, but mostly indifferent
+* 0: Indifferent
+* -0: Mildly against, but mostly indifferent
+* -1 : Strongly against. This is expected to be accompanied by a reason.
+
+For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. 
+
+## Contentious decisions
+
+Should a decision:
+
+* Be strategic or impact many within the project,
+* Or if there is contention in deciding, and after discussion at least one member is still strongly against the motion, by continuing to vote -1,
+
+then the motion should be referred to the PSC (via our email list).
+
+For contentious motions presented to the PSC, a motion is considered passed when there are more PSC +1 votes than -1 votes, and either all PSC members have voted, or 4 days have passed. In the case of a tie, the PSC chair shall have 1.5 votes.
+
+## Record decisions
+
+All decisions should be forwarded to our email list to provide the rest of the community with the opportunity to vote. 
```

Comment:
Does this mean that a decision passed by the PSC could potentially be voted down by the community?

Reply:
Short answer is "no, community doesn't out rank the PSC". But the fact that you found the message confusing means that I should go back and reword to remove the ambiguity. I'll have a crack at rewording over next few days.